### PR TITLE
Tune lander shader shadows to use front-face shadow maps.

### PR DESCRIPTION
**Pull request by mogumbo**
| Jira Ticket 🎟️ | [OCEANWATER-XXX](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-539) |

## Summary of Changes
This work is in support of this other PR: https://github.com/nasa/ow_europa/pull/18

### CODE REVIEW

File: ow_lander/materials/scripts/ow_lander.frag [hunk 1]
```diff
@@ -184,10 +186,18 @@ void lighting(vec3 wsDirToSun, vec3 wsDirToEye, vec3 wsNormal, vec4 wsDetailNorm
   const float specular_power = 100.0;
 
   // shadows
+  // Compute shadow lookup bias using formula from
+  // http://www.opengl-tutorial.org/intermediate-tutorials/tutorial-16-shadow-mapping/
+  // Should be able to bake this bias into the shadow map using constant_bias
+  // and slope_scale_bias in IRGShadowParametersPlugins, but it doesn't work.
+  float constantBias = vsPos.z * 0.00001 - 0.00002; // Normally a constant, but this works better with PSSM
```
Comment (AstroStucky) [inline ow_lander/materials/scripts/ow_lander.frag:193]: I have the same comment on this variable name as I did in the other PR.

**Review by AstroStucky — APPROVED** [review]
Looks good. Test description left in the other PR.

**Review by kmdalal — APPROVED** [review]
I did the same test that @AstroStucky describes in the corresponding ow_europa PR.  Looks good!  Not familiar with this code and its format, so no comments here.

File: ow_lander/materials/scripts/ow_lander.frag (hunk 1 above)
Comment (mogumbo) [reply ow_lander/materials/scripts/ow_lander.frag:193]: I debated this name too. In the end, I think the comment explains it well enough and the name constantBias will make it easier to reconcile with any shadow bias literature you read.

**Pull request merged**